### PR TITLE
Fix spelling issues in GSdx Linux configuration window

### PR DESCRIPTION
**Pull request by Gareth422**

### CODE REVIEW

File: plugins/GSdx/GSLinuxDialog.cpp [hunk 1]
```diff
@@ -453,7 +453,7 @@ bool RunLinuxDialog()
 	// The main area for the whole dialog box.
 	GtkWidget* main_box    = gtk_vbox_new(false, 5);
 	GtkWidget* central_box = gtk_vbox_new(false, 5);
-	GtkWidget* advance_box = gtk_vbox_new(false, 5);
+	GtkWidget* advanced_box = gtk_vbox_new(false, 5);
```
Comment (gregory38) [inline plugins/GSdx/GSLinuxDialog.cpp]: Please (vertical) realign the code properly

File: plugins/GSdx/GSLinuxDialog.cpp [hunk 2]
```diff
@@ -488,9 +488,9 @@ bool RunLinuxDialog()
 
 	// Handle some nice tab
 	GtkWidget* notebook = gtk_notebook_new();
-	gtk_notebook_append_page(GTK_NOTEBOOK(notebook), central_box, gtk_label_new("Global Setting"));
-	gtk_notebook_append_page(GTK_NOTEBOOK(notebook), advance_box, gtk_label_new("Advance Setting"));
-	gtk_notebook_append_page(GTK_NOTEBOOK(notebook), debug_box  , gtk_label_new("Debug/Recording Setting"));
+	gtk_notebook_append_page(GTK_NOTEBOOK(notebook), central_box, gtk_label_new("Global Settings"));
```
Comment (gregory38) [inline plugins/GSdx/GSLinuxDialog.cpp]: Same here, realign the code.

File: plugins/GSdx/GSLinuxDialog.cpp [hunk 3]
```diff
@@ -467,8 +467,8 @@ bool RunLinuxDialog()
 	GtkWidget* hw_table     = CreateTableInBox(central_box , "Hardware Mode Settings"               , 7  , 2);
 	GtkWidget* sw_table     = CreateTableInBox(central_box , "Software Mode Settings"               , 2  , 2);
 
-	GtkWidget* hack_table   = CreateTableInBox(advance_box , "Hacks"                                , 7 , 2);
-	GtkWidget* gl_table     = CreateTableInBox(advance_box , "OpenGL Very Advanced Custom Settings" , 6 , 2);
+	GtkWidget* hack_table   = CreateTableInBox(advanced_box , "Hacks"                                , 7 , 2);
```
Comment (gregory38) [inline plugins/GSdx/GSLinuxDialog.cpp]: remove the space before , so code is realigned.

File: plugins/GSdx/GSLinuxDialog.cpp [hunk 4]
```diff
@@ -9,7 +9,6 @@
  *
  *  This Program is distributed in the hope that it will be useful,
  *  but WITHOUT ANY WARRANTY; without even the implied warranty of
```
Comment (rz5) [inline plugins/GSdx/GSLinuxDialog.cpp]: @Gareth422, I believe you accidentally deleted a line from this file's license text. Ya might want to fix that.

**Comment by gregory38** [comment]
I squashed your commit and merged them.

**Pull request closed without merging**